www/front_src/src/Resources/colors.ts

```diff
@@ -0,0 +1,19 @@
+import { lime, purple } from '@material-ui/core/colors';
+
+const downtimeColor = purple[500];
```

Comment:
I'm currently adding chose colors in the theme in another branch. So we'll update this once PR is merged